data/processed/ref/mean_ref_stem.yaml

```diff
@@ -0,0 +1,23 @@
+api: fetch
+api_documentation: https://www.ref.ac.uk/2014/pubs/201401/
+description: These are the results of the Research Excellence Framework, where university departments in various disciplines are assessed on the quality of their research. The latest REF was conducted in 2014 so data is available only for one year
+endpoint_static: https://results.ref.ac.uk/(S(hlvnuqzwkag44jp3df3d4q14))/DownloadFile/AllResults/xlsx
+framework_group: public_rnd_capability
+is_experimental: False
+nuts_level: 2
+source: the data provider: https://results.ref.ac.uk/
+data_date: 20200204
+year_range: [2014, 2014]
+schema:
+  nuts_id:
+    type: NutsRegion.id
+  nuts_year_spec:
+    type: NutsRegion.year_spec
+  value:
+    description: The mean REF score for the NUTS2 area considering only STEM subjects (see aux/ref_stem.txt) for the subjects that we selected. This is the average of the scores in all departments weighted by the Full time equivalents submitted in each category (4*, 3*, 2* with higher scores representing better assessments)
+    id: mean_ref_stem
+    label: Mean Research Excellence Framework Score
+    type: ref_score
```

Comment:
Please use `REF`

Reply:
Also please add a `format`, here or to the `REF` type itself if it makes sense as it could be a float.